core/statistics/machine/cpuStatistics.go

```diff
@@ -21,12 +23,19 @@ func (cs *CpuStatistics) ComputeStatistics() {
 		return
 	}
 
-	cpuUsagePercent, err := currentProcess.Percent(durationSecond)
+	cpuUsagePercent, err := currentProcess.CPUPercent()
+	if err != nil {
+		cs.setZeroStatsAndWait()
```

Comment:
log error.

Reply:
i think is not need to log and error here because is not critical if cannot read CPU Percent.